main.nf

```diff
@@ -273,6 +287,7 @@ process MULTIQC_PER_FLOWCELL {
     script:
     threshold_parameter = qc_thresholds ? "-c ${qc_thresholds}" : ""
     """
+    cp rRNA/rrna_table.tsv rRNA/rrna_plot.tsv
```

Comment:
I saw that you explained this here: https://github.com/Molmed/seqreports/pull/44#discussion_r650709694 . I think it would be nice to add a comment with an explanation to why this copying step is needed and perhaps a link to a MultiQC issue, if it has been reported as a bug / not desired feature.